### PR TITLE
Feature Store ingestion streaming notebook

### PR DESCRIPTION
**Pull request by soheilazangeneh**
**REQUIRED:** Add a summary of your PR here, typically including why the change is needed and what was changed. Include any design alternatives for discussion purposes.

The streaming ingestion feature for Feature Store allows developers to write Features to their Feature Store within milliseconds using the underlying [WriteFeatureValues API](https://github.com/googleapis/python-aiplatform/blob/main/google/cloud/aiplatform/preview/featurestore/entity_type.py) compared to the ~4-5 minutes it takes to do a write using the ImportFeatureValues API (more info [here](https://docs.google.com/document/d/1aXZW4BxojW9ICjanWmUoiCT1DxR-8nN4SUVkhe-a6bg/edit#)). The streaming ingestion feature has been introduced to the Vertex SDK under the preview namespace and requires a Python notebook and a code sample highlighting how to use the feature in the Vertex SDK.

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/feature_store/feature_store_streaming_ingestion_sdk.ipynb [hunk 1]
```diff
@@ -0,0 +1,772 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/official/feature_store/feature_store_streaming_ingestion_sdk.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #17.        "body_mass_g": {"value_type": "DOUBLE"},
</pre><p>format body_mass_g and sex like the above features</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1321/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/feature_store_streaming_ingestion_sdk.ipynb:1]: <p>we -&gt; you</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1321/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/feature_store_streaming_ingestion_sdk.ipynb:1]: <p>Vertex SDK -&gt; Vertex AI SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1321/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

[the remark this answers is not in the document]
Comment (nayaknishant) [reply notebooks/official/feature_store/feature_store_streaming_ingestion_sdk.ipynb:1]: <p>I think it should be "Streaming ingestion" instead: https://cloud.google.com/vertex-ai/docs/featurestore/ingesting-stream</p><p> </p>

[the remark this answers is not in the document]
Comment (nayaknishant) [reply notebooks/official/feature_store/feature_store_streaming_ingestion_sdk.ipynb:1]: <p>Does this line need a heading? Would the heading here be "Check if EntityType created"?</p>

Comment (nayaknishant) [inline notebooks/official/feature_store/feature_store_streaming_ingestion_sdk.ipynb:1]: <p>nice! :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1321/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (nayaknishant) [inline notebooks/official/feature_store/feature_store_streaming_ingestion_sdk.ipynb:1]: <p>entity type called penguin -&gt; entity type named penguin_entity_type</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1321/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

[the remark this answers is not in the document]
Comment (andrewferlitsch) [reply notebooks/official/feature_store/feature_store_streaming_ingestion_sdk.ipynb:1]: I would use Verify that EntityType is created

[the remark this answers is not in the document]
Comment (soheilazangeneh) [reply notebooks/official/feature_store/feature_store_streaming_ingestion_sdk.ipynb:1]: <p>Updated.</p>

[the remark this answers is not in the document]
Comment (soheilazangeneh) [reply notebooks/official/feature_store/feature_store_streaming_ingestion_sdk.ipynb:1]: <p>Done.</p>

[the remark this answers is not in the document]
Comment (soheilazangeneh) [reply notebooks/official/feature_store/feature_store_streaming_ingestion_sdk.ipynb:1]: <p>Done.</p>

[the remark this answers is not in the document]
Comment (soheilazangeneh) [reply notebooks/official/feature_store/feature_store_streaming_ingestion_sdk.ipynb:1]: <p>Done.</p>

[the remark this answers is not in the document]
Comment (soheilazangeneh) [reply notebooks/official/feature_store/feature_store_streaming_ingestion_sdk.ipynb:1]: <p>Done.</p>

Comment (soheilazangeneh) [reply notebooks/official/feature_store/feature_store_streaming_ingestion_sdk.ipynb:1]: <p>Done.</p>

[the remark this answers is not in the document]
Comment (soheilazangeneh) [reply notebooks/official/feature_store/feature_store_streaming_ingestion_sdk.ipynb:1]: <p>Done.</p>

Comment (soheilazangeneh) [reply notebooks/official/feature_store/feature_store_streaming_ingestion_sdk.ipynb:1]: <p>Good catch! Done.</p>

Comment (soheilazangeneh) [reply notebooks/official/feature_store/feature_store_streaming_ingestion_sdk.ipynb:1]: <p>Done.</p>

Comment (soheilazangeneh) [reply notebooks/official/feature_store/feature_store_streaming_ingestion_sdk.ipynb:1]: <p>Done.</p>

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**